config-files/aggregator.yaml

```diff
@@ -11,3 +11,6 @@ rollup_ids_to_rpc_urls:
   2: ws://localhost:8546
 rollup_ids_to_registry_addresses:
   2: 0x0000000000000000000000000000000000000000
+
+# metrics related
```

Comment:
Shouldn't `enable_metrics` be here as well?

Reply:
?

Reply:
done